openshift/patches/use_quay_images.patch

```diff
@@ -0,0 +1,22 @@
+diff --git a/test/scripts/first-event-delay.sh b/test/scripts/first-event-delay.sh
+index f4d4f3be3..dc132a6c3 100755
+--- a/test/scripts/first-event-delay.sh
++++ b/test/scripts/first-event-delay.sh
+@@ -72,7 +72,7 @@ spec:
+     spec:
+       containers:
+       - name: event-display
+-        image: ko://knative.dev/eventing/cmd/event_display
++        image: quay.io/openshift-knative/knative-eventing-sources-event-display:v0.13.2
+ ---
+ apiVersion: sources.knative.dev/v1
+ kind: SinkBinding
+@@ -114,7 +114,7 @@ spec:
+     spec:
+       containers:
+       - name: single-heartbeat
+-        image: ko://knative.dev/eventing/test/test_images/heartbeats
++        image: quay.io/openshift-knative/knative-eventing-sources-heartbeats:v0.13.2
```

Comment:
sure we wanna use that images? (version)

Reply:
Yes, we tested this script with this version.

I don't want to verify it with newer versions for now.